.github/workflows/abi_bindings_checker.yml

```diff
@@ -27,6 +27,12 @@ jobs:
       - name: Install Foundry
         run: ./scripts/install_foundry.sh
 
+      - name: Install solc
+        run: |
+          wget https://github.com/ethereum/solidity/releases/download/v0.8.25/solc-static-linux
```

Comment:
Could we move the solidity version used to a more visible place? Or ideally keep it defined in a single place between both `solc` and `forge`

Reply:
Addressed in https://github.com/ava-labs/teleporter/pull/626